pyicloud/services/photos.py

```diff
@@ -180,11 +180,15 @@ def albums(self):
                 ):
                     continue
 
-                self._construct_album(folder)
+                if folder["fields"]["albumType"]["value"] == 0:
```

Comment:
I see what you did here. And I really do not like it. 😂
I'm just not a fan of "magical numbers" being used throughout the code as they are hard to memorize.

How about something like this:

folder_types = {"album":0, "folder": 3}

if folder["fields"]["albumType"]["value"] == folder_types["album"]:
   ...

That makes it easy to read and the logic becomes clearer.